doc/deployment/deploy_custom.md

```diff
@@ -1,6 +1,118 @@
 # Custom Deployments
```

Comment:
Hi level comment:

All of the information you have here is good, but I think we can improve the information architecture a bit. It feels like you lost the forest for the trees given that this has been an ongoing side project for so long. The 2nd half of the doc is excellent, but the first half isn't clear where it fits in the overall process and other deploy docs:

E.g. 
1. On_premises doc
2. Custom deploy doc (this one)
3. ? --  Is there a 3rd?

We probably just need to put a more organized opening, TOC, etc.

Reply:
We chatted about this on the phone, and I also discussed with @svekars.  Every doc needs to be somewhat self-contained, with preambles that bring the reader up to speed.